### DISCUSSION TITLE
how to translate datatset title and description?

### DISCUSSION THREAD

**Discussion by kumarvivek1752** [ckan/ckan#7854] [Q&A]
hi I'm trying to translate the `dataset name ` and `description` using `ckanext-fluent`
but it not updating fields in `http://localhost:5000/dataset/new` any one have idea or guide to do this
![image](https://github.com/ckan/ckan/assets/67893205/2c470f9b-dbea-4820-acb7-d004c46d4f63)

**Comment by muhammetcetinyamac** [comment]
Hi, You can try using the 'ckanext-multilang' extension to localize datasets and resources. 
[https://github.com/geosolutions-it/ckanext-multilang](https://github.com/geosolutions-it/ckanext-multilang)
